docs/FunctionalTestBuildScripts.md

```diff
@@ -0,0 +1,29 @@
+# Build Scripts
+
+This file records scripts used to compile the test files, in alphabetical order.
+Base scenario is a simple hello world program built with different parameters for testing purpose.
+Test files are located in [BaselineTestsData](https://github.com/microsoft/binskim/tree/main/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData) and [FunctionalTestsData](https://github.com/microsoft/binskim/tree/main/src/Test.FunctionalTests.BinSkim.Rules/FunctionalTestsData).
+
+## Native_x64_VS2019_CPlusPlus_GsDriverEntry_And_UserFnUseSafeBuffers.exe
```

Comment:
>Native_x64_VS2019_CPlusPlus_GsDriverEntry_And_UserFnUseSafeBuffers.exe [](http://example.com/codeflow?start=3&length=70)

explain for the 4 test files:

case of: 
1. only GsDriverEntry that is in the allow list  ---- result should be pass
2. only user functions not in the allow list (I added 2 functions)  ---- result should be fail
3. GsDriverEntry  + user functions  ---- result should be fail
4. None  ---- result should be pass